server/tests/http/formationsRoutes-test.js

```diff
@@ -1,10 +1,15 @@
 const assert = require("assert");
 const config = require("../../src/config");
-const { startServer } = require("../utils/testUtils");
 const { mockInserJeunesApi } = require("../utils/apiMocks");
 const { insertFormationsStats } = require("../utils/fakeData");
 
 describe("formationsRoutes", () => {
+  let httpClient;
+  before(async () => {
```

Comment:
Moi je préfère quand le test est auto portant et qu'il contient dans le `it` tous les elements lui permettant de s’exécuter. L'autre inconvenient du `before` c'est qu'il n'est pas possible d’empêcher l'execution du `before`  pour définir un contexte différent

Bon en l'occurence ca ne pose pas de pbl donc ok

Reply:
Non ok laissons comme ca